be/src/runtime/sender_queue.cpp

```diff
@@ -536,7 +536,9 @@ Status DataStreamRecvr::PipelineSenderQueue::add_chunks_and_keep_order(const PTr
 void DataStreamRecvr::PipelineSenderQueue::decrement_senders(int be_number) {
     {
         std::lock_guard<Mutex> l(_lock);
-        if (_sender_eos_set.find(be_number) != _sender_eos_set.end()) {
+        if (UNLIKELY(_sender_eos_set.find(be_number) != _sender_eos_set.end())) {
+            LOG(ERROR) << "More than one EOS from " << be_number << " in fragment "
```

Comment:
Our RPC has resend, it is normal to have this scenario, is it necessary to print ERROR here?

Reply:
the EOS should be sent at most exactly once.